app_flutter/lib/state/flutter_build.dart

```diff
@@ -11,6 +11,8 @@ import 'package:cocoon_service/protos.dart' show Commit, CommitStatus, RootKey,
 import '../service/cocoon.dart';
 import '../service/google_authentication.dart';
 
+import 'brooks.dart';
```

Comment:
Should this be grouped in the above import block? Or do imports in the CWD get grouped in their own import block?

Reply:
Good point, will fix.